Makefile

```diff
@@ -137,16 +130,9 @@ integration_xml: clean generate
 	go test -tags='$(GO_TAGS)' --json ./pkg/... ./cmd/... -coverprofile cover.out > integration-tests.json
 	gotestsum --junitfile integration-tests.xml --raw-command cat integration-tests.json
 
-check-fmt:
```

Comment:
Why are we removing this?

Reply:
golangci-lint has go-imports and vet built-in so `check-fmt` is redundant.